packages/oui-popover/src/popover.directive.js

```diff
@@ -9,6 +9,7 @@ export default () => {
             text: "@ouiPopover",
             title: "@?",
             placement: "@?ouiPopoverPlacement",
+            scope: "=?ouiPopoverScope",
```

Comment:
Why not `<` ?

Reply:
Yes, no utility to have a two way here. Updated